.github/scripts/install-torch-tensorrt-windows.sh

```diff
@@ -5,8 +5,9 @@ source ${BUILD_ENV_FILE}
 export EXTRA_INDEX_URL="https://download.pytorch.org/whl/test/${CU_VERSION}"
 # Install all the dependencies required for Torch-TensorRT
 ${CONDA_RUN} pip install --pre -r ${PWD}/tests/py/requirements.txt --use-deprecated=legacy-resolver --extra-index-url=${EXTRA_INDEX_URL}
+${CONDA_RUN} pip install --pre -r ${PWD}/tests/py/requirements.txt --use-deprecated legacy-resolver
```

Comment:
Is this duplicated change ?

Reply:
Yes, will remove this - thanks for the catch